docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,642 @@
+        </privileges>
+    </manifest>
+    ```
+
+2.  Sending launch request
+
+    You can send launch request using app_control_h. The app_control_h will contains the Component-Based Application's ID and the component ID. The component ID is optional, if you do not set the component ID, then the main component instance will be created.
+
+    You can launch a Component-Based application as follow.
+    ```
+    static int __app_control_send(const char *app_id, const char *component_id,
+            void *user_data)
+    {
+        app_control_h handle = NULL;
+        int ret;
+
+        ret = app_control_create(&handle);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        ret = app_control_set_app_id(handle, app_id);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        if (component_id) {
+            ret = app_control_set_component_id(handle, component_id);
+            if (ret != APP_CONTROL_ERROR_NONE) {
+                app_control_destroy(handle);
+                return ret;
+            }
+        }
+
+        ret = app_control_set_launch_mode(handle,
+                APP_CONTROL_LAUNCH_MODE_GROUP);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_send_launch_request_async(handle,
+                __app_control_result_cb, NULL, user_data);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        return 0;
+    }
+
+    static void __button_clicked_cb(void *data, Evas_Object *obj,
+        void *event_info) {
+        __app_control_send("org.tizen.base-component", "base-frame", NULL);
+    }
+    ```
+
+<a name="group_launch"></a>
+## Group Launching Management
+
+The Component-Based Application also provides an [application group feature](../app-management/app-controls.md#group). You can use it as follows.
+
+1. Declaring launch privilege in the manifest file
+    ```
+    <component-based-application appid="org.tizen.base-component" exec="@BINDIR@/base-component" nodisplay="false" multiple="false" type="capp">
+        <frame-component id="base-frame" launch_mode="caller" main="true" icon-display="false" taskmanage="true">
+          <icon>org.tizen.sample.png</icon>
+          <label>FrameComponent</label>
+          <label xml:lang="en-us">FrameComponent</label>
+          <label xml:lang="ko-kr">FrameComponent[KOR]</label>
+        </frame-component>FrameComponent</label>
+        <icon>FrameComponent.png</icon>
+      </component-based-application>
+      <privileges>
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+      </privileges>
+    </manifest>
+    ```
+
+2.  Sending launch request using component_h
+
+    You have to send launch request with component_h so that application framework can tell which component instance request group launch.
```

Comment:
Doubt:
Are we saying:
You have to send a launch request with `component_h` so that the application framework can determine which component instance request group is launched.

Please verify and share your opinion, so that we can rephrase it if needed.

Reply:
I changed it to

You have to send a launch request with component_h so that the application framework can determine which component instance request launch an application